mewarpx/examples/thermionic_ignition.py

```diff
@@ -0,0 +1,202 @@
+from mewarpx import util as mwxutil
+
+from picmistandard import simulation
+mwxutil.init_libwarpx(ndim=2, rz=False)
+
+from minerva import util as minutil
+
+from pywarpx import picmi
+
+from mewarpx import assemblies, emission, mepicmi
+
+from mewarpx.mwxrun import mwxrun
+from mewarpx.diags_store import diag_base
+
+constants = picmi.constants
+
+####################################
+# physics parameters
+####################################
+
+P_INERT = 2.0 # torr
+T_INERT = 300.0 # K
+N_INERT = (P_INERT * minutil.torr_cgs) / (minutil.kb_cgs * T_INERT) # m^-3
```

Comment:
Should we remove the dependency on minerva from this file and just hardcode this in for now so that it's more compatible with memaster?